rq/worker.py

```diff
@@ -410,6 +410,8 @@ def work(self, burst=False):
 
                     result = self.dequeue_job_and_maintain_ttl(timeout)
                     if result is None:
+                        if burst:
+                            self.log.info("RQ worker, '%s', has died." % self.key)
```

Comment:
Can you change this to "shutting down" instead of has died? "Has died" sounds like something unexpected has happened to the worker.